docker_dyno_driver.go

```diff
@@ -96,6 +98,11 @@ func (dd *DockerDynoDriver) Start(ex *Executor) error {
 		log.Fatal(err)
 	}
 
+	ex.container, err = dd.d.c.InspectContainer(container.ID)
```

Comment:
we should try for a pre-defined (maybe configurable?) timeout. Or you could poll the docker API waiting until the container is up before grabbing its IP.